src/phantom.js

```diff
@@ -82,7 +82,7 @@ function ResourceHandler(htmlroot, isWindows, resolve) {
             }
             networkRequest.changeUrl(url);
         } else if (ignoreRequests.test(url)) {
-            networkRequest.abort();
+            networkRequest.cancel();
```

Comment:
Don't know when I'll get to investigate personally. @XhmikosR did you try setting L61 to something like:
```
this.on('onResourceRequested', true, function (requestData, networkRequest) {
```
and L87 to `});` ?